ext/configuration.h

```diff
@@ -94,6 +94,8 @@ enum ddtrace_dbm_propagation_mode {
     CONFIG(STRING, DD_TRACE_MEMORY_LIMIT, "")                                                                  \
     CONFIG(BOOL, DD_TRACE_REPORT_HOSTNAME, "false")                                                            \
     CONFIG(BOOL, DD_TRACE_FLUSH_COLLECT_CYCLES, "false")                                                       \
+    CONFIG(BOOL, DD_TRACE_REMOVE_ROOT_SPAN_LARAVEL_QUEUE, "false")                                              \
```

Comment:
If we default this to false, there's little value in it? And we could just tell the customer to set DD_TRACE_GENERATE_ROOT_SPAN instead of this option.